packages/cmr-client/package.json

```diff
@@ -35,11 +38,16 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@cumulus/aws-client": "14.0.0",
+    "@cumulus/common": "14.0.0",
     "@cumulus/errors": "14.0.0",
     "@cumulus/logger": "14.0.0",
     "got": "^11.8.5",
     "lodash": "^4.17.21",
     "public-ip": "^3.0.0",
-    "xml2js": "^0.4.19"
+    "xml2js": "^0.4.19",
+    "zod": "^3.20.2"
+  },
+  "devDependencies": {
+    "jose": "^4.12.0"
```

Comment:
Is there any reason to use `jose` instead of the existing package `jsonwebtoken` we have?